bobtemplates/plone/buildout/buildout.cfg.bob

```diff
@@ -42,4 +42,15 @@ recipe = zc.recipe.testrunner
 defaults = ['--auto-color', '--auto-progress']
 eggs =
     Plone
-    ${buildout:eggs}
\ No newline at end of file
+    ${buildout:eggs}
+
+[zopepy]
+recipe = zc.recipe.egg
+eggs =
+   Products.CMFPlone
+   ${instance:eggs}
+   ${test:eggs}
```

Comment:
I don't think we need the test-eggs here

Reply:
for autocompletion test-eggs meight be usefull too, so i would keep it in here.

Reply:
Do we need test-eggs then? @jensens @MrTango